src/components/v2/V2Create/StartOverButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { t, Trans } from '@lingui/macro'
+import { Button, Modal } from 'antd'
+import { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+
+import { editingV2ProjectActions } from 'redux/slices/editingV2Project'
+
+export default function StartOverButton() {
```

Comment:
How do you feel about moving away from default imports? The big issue with defaults I find is if the name changes, it is not obvious for importers that it has changed and relies on the refactor tool working correctly; if you use named exports, the code won't compile, forcing you to change it.

Reply:
Right right. It's a great point that I've never considered. Would love to hear @aeolianeth's thoughts on the matter

Reply:
SGTM